extensions/wikia/Forum/ForumController.class.php

```diff
@@ -356,4 +359,23 @@ public function forumRelatedThreads() {
 			$this->response->setVal( 'messages', $messages );
 		}
 	}
+
+	public function messageTopic() {
+		// stub function
+	}
+
+	private function addAssets() {
+		JSMessages::enqueuePackage( 'Wall', JSMessages::EXTERNAL );
+		$this->response->addAsset( 'forum_js' );
+		$this->response->addAsset( 'extensions/wikia/Forum/css/ForumBoard.scss' );
+		$this->response->addAsset( 'extensions/wikia/Wall/css/MessageTopic.scss' );
+
+		// VOLDEV-36: separate monobook styling
+		if ( $this->app->checkSkin( 'monobook' ) ) {
+			$this->response->addAsset( 'extensions/wikia/Forum/css/monobook/ForumMonobook.scss' );
+			$this->response->addAsset( 'extensions/wikia/Forum/css/monobook/ForumBoardMonobook.scss' );
+		}
+
+		$this->addMiniEditorAssets();
```

Comment:
It does not. It is used only once and all it does is a side effect. It is not something we can test in a sane way and I do not think a separare `addMiniEditorAssets()` makes `addAssets()` less or more testable. It is simply a piece of code written for IDE rather than for a human. Pointless. :(